app/assets/stylesheets/partials/_event_box.scss

```diff
@@ -0,0 +1,51 @@
+.event-box {
+    width: 350px;
+    padding: 2.1rem 1.5rem;
+
+    border: 1px solid $v3-color_green-light;
+
+    * { color: $color_font-light !important; }
+
+    &__header {
+        display: flex;
+        align-items: center;
+
+        h2 {
+            margin: 0;
+
+            font-size: $font_size-xlarge;
```

Comment:
Her design uses 20px font size as the base whereas websites use 16px, so all sizing translations need to be scaled to 80%. Thus (44px * .8) / 16px = 35.2px / 16px = 2.2rem. 

I use 2.1875rem for `$font_size-xlarge` because I rounded to 35px since some browsers have trouble with subpixel rendering.

Reply:
Ok, that makes sense. The only thing is wondering how we should approach the transition to 20px when some of the older pages are still going to be base 16px. In my other PR I just assumed a 16px base and did calculations to scale things up to 20px on the new pages, but am open to ideas on this.

Reply:
Browsers have limited space while artboards are virtualized and have unlimited space which makes her pixel base arbitrary. Converting a design to browser standards, with 16px being this particular standard, will translate her ideas more sustainably.

Reply:
Otherwise everything will just look really squished and zoomed in.

Reply:
The reason I didn't think it was arbitrary is that Adobe XD provides for virtual resolutions in its art boards that are specific to devices:

<img width="498" alt="image" src="https://user-images.githubusercontent.com/565647/54766852-6cacf380-4bd2-11e9-858b-e22ba01642c9.png">

And in this case we're using 1920 for web. So my understanding is an intentional designer in XD should be setting font-px to what they want the developers to use in the CSS. I based the idea of a 1 to 1 adobe XD pixel to CSS pixel mapping on the forum post from Adobe support at https://forums.adobe.com/thread/2140211. 

That being said given the mismatch between the old and new design, and the fact 20px isn't standard, open to the idea that this was arbitrary so we might want to check with the designer first and scale this down to 16px.